pkg/model/permissions.go

```diff
@@ -0,0 +1,33 @@
+package model
+
+type PermissionGrantAccess string
+
+const (
+	PermissionGrantAccessAll  PermissionGrantAccess = "all"
+	PermissionGrantAccessSome PermissionGrantAccess = "some"
+	PermissionGrantAccessNone PermissionGrantAccess = "none"
+)
+
+type PermissionName string
+
+const (
+	PermissionNameWorkflowCreate  PermissionName = "workflow.create"
+	PermissionNameWorkflowDelete  PermissionName = "workflow.delete"
+	PermissionNameWorkflowEdit    PermissionName = "workflow.edit"
+	PermissionNameWorkflowRun     PermissionName = "workflow.run"
+	PermissionNameWorkflowView    PermissionName = "workflow.view"
+	PermissionNAmeWorkflowApprove PermissionName = "workflow.approve"
+)
+
+type PermissionSummary struct {
+	Name PermissionName `json:"name"`
+}
+
+type PermissionGrant struct {
+	Grant      PermissionGrantAccess `json:"granted"`
+	Permission PermissionSummary     `json:"permission"`
+}
+
+type EntityAccess struct {
+	PermissionGrants *[]PermissionGrant `json:"permission_grants"`
+}
```

Comment:
This isn't used as far as I can tell. Can we jettison it?

Reply:
I suppose. I could see us one day listing the possible operations a user is allowed to perform on a resource

Reply:
Yeah let's add it in when it's relevant! Less confusing than carrying it around and it not doing anything.